python/pyarrow/__init__.py

```diff
@@ -205,7 +228,7 @@ def get_library_dirs():
     # are not shipped inside the pyarrow package (see also ARROW-2976).
     from subprocess import call, PIPE, Popen
     pkg_config_executable = _os.environ.get('PKG_CONFIG', None) or 'pkg-config'
-    for package in ["arrow", "plasma", "arrow_python"]:
+    for package in ["arrow", "arrow_gpu", "plasma", "arrow_python"]:
```

Comment:
what happens if this fails?

Reply:
hmm, when arrow_gpu is not available, it will be ignored. On the other hand, arrow and arrow_gpu libraries should be located in the same directory. Suggest dropping "arrow_gpu" from the list. Thoughts?

Reply:
Removed arrow_gpu. pkg-config approach needs a separate test setup.